data/applications.yaml

```diff
@@ -92,3 +92,16 @@ supported_os: [Windows]
 urls:
 - 'http://www.hexacorn.com/blog/2012/09/16/beyond-good-ol-run-key-part-2/'
 - 'http://acritum.com/software/manuals/winrar/html/helpcommandsvirusscan.htm'
+---
+name: MicrosoftSqlServerErrorLogs
+doc: Microsoft SQL Server's error log files.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%environ_programfiles%%\Microsoft SQL Server\*\MSSQL\LOG\ERRORLOG*'
+    - '%%environ_programfilesx86%%\Microsoft SQL Server\*\MSSQL\LOG\ERRORLOG*'
+    separator: '\'
+labels: [Software, Logs]
+supported_os: [Windows]
+urls: ['https://github.com/EricZimmerman/KapeFiles/blob/a60491344a6c24ad1e6987f66283da960b9be739/Targets/Logs/MSSQLErrorLog.tkape']
```

Comment:
Please link to a source that is more informative about the artifact not some other definition format

Reply:
maybe create a forensics wiki article for this artifact? or leave the link out? seems to be fairly self explanatory the currently linked article adds no value

Reply:
Good point, I'll leave the link out. It was mainly to give the credits to the people who I copied the information from. Thanks :)

Reply:
then I opt to create a forensics wiki article, typically the credit does not need go to a single person or project, eg the file you link to seems to be authored by someone else than the project maintainer, but information about the error log is something that likely originates from Microsoft https://docs.microsoft.com/en-us/sql/relational-databases/performance/view-the-sql-server-error-log-sql-server-management-studio?view=sql-server-ver15